canonical-websites/cn.ubuntu.com
# Add the global navigation component

## Done
Add the global navigation component

## QA
- Pull down this branch
- Run `./run clean`
- Run `./run`
- Go to http://0.0.0.0:8010/
- Check that the global navigation works as usual

## Issue / Card
Fixes https://github.com/ubuntudesign/vanilla-squad/issues/75
